### ISSUE TITLE
Javascript missing the same level of documentation as other clients

### ISSUE THREAD

**Issue by jamesatha** [algorithmiaio/dev-center#128]
It seems like it's lacking a few sections. And I think there was a reason for that. Can we explain these reasons so at least people know why

**Comment by StephanieKim** [comment]
Yeah, you're right - we don't support the data api in javascript and we should say that for sure!
